packages/integrations/web-vitals/src/index.ts

```diff
@@ -2,7 +2,8 @@ import { defineDbIntegration } from '@astrojs/db/utils';
 import { AstroError } from 'astro/errors';
 import { WEB_VITALS_ENDPOINT_PATH } from './constants.js';
 
-export default function webVitals() {
+export default function webVitals({ deprecated }: { deprecated?: boolean } = {}) {
+	process.env.DEPRECATE_WEB_VITALS = deprecated ? 'true' : undefined;
```

Comment:
Shouldn't we set this only when running a `db push`? Or is it intentional?

Reply:
I guess you might want it in `dev` etc. too? If you happened to run it while deprecated. Probably doesn’t matter either way though — I just wanted it to match how a user-defined table might behave really.